src/api/models/lines.js

```diff
@@ -99,31 +107,35 @@ lineSchema.methods.updateStatus = async function(status) {
     }
 
     if (msg !== null) {
-        return msg;
+        return Promise.reject(msg);
```

Comment:
what functionality does Promise.reject have when u step out of the function? do .catch statements catch rejected promises?

Reply:
I added try/catch to updateLineStatus (the patch route) since it lacked it, and also to answer your question.

Here is the code:
![image](https://user-images.githubusercontent.com/35344365/37002132-aa0534d0-2096-11e8-9605-0e09b46ecdcf.png)

And here is the result, trying to send a line from notification to startRecruiter. Obviously, this is disallowed, so the promise rejects, just as the code you highlighted showed. This is caught by the code in the above screenshot, and you can see that the client is returned the message that the promise rejected with:
![image](https://user-images.githubusercontent.com/35344365/37002021-547dd3c8-2096-11e8-92b7-c50e3bfab252.png)